### ISSUE TITLE
"Please wait for all operations to finish", never finishes

### ISSUE THREAD

**Issue by cjallen88** [zealdocs/zeal#990]
"Please wait for all operations to finish" message appears when attempting to leave the Docsets window after installing some docsets. Many of these were installed concurrently. All operations that I can see have finished. 

![image](https://user-images.githubusercontent.com/7375123/45422880-86043e80-b689-11e8-8a46-846feec9dfd6.png)

**Comment by cjallen88** [comment]
Also should mention this is the latest windows 64bit MSI available from https://zealdocs.org/download.html as of today

**Comment by trollixx** [comment]
This usually indicates that some of the docsets take longer time installing. Are you sure nothing is missing on the screen shot?

**Comment by cjallen88** [comment]
Hi, all loading bars on the 'Available' tab were complete and no longer visible, it did eventually let me close the window after another 10 or so minutes as if some post-processing was taking place that was not included in the loading bars.

**Comment by Giancarlos** [comment]
This is still an issue in 0.6.1 where there is no progress bar or any indication that Zeal is still installing docsets. The progress bars disappear once download is complete, but I guess during install it just doesn't show any progress.

**Comment by trollixx** [comment]
Duplicate of #707.
